### PR TITLE
FIX : minor ui changes solved #447

### PR DESCRIPTION
**Pull request by mohitparmar1**
### PR Fixes:
Fix minor UI bug affecting visibility of specific numbers in light mode

resolves issue #447 

##Screenshots
Screenshot 2024-04-17 064636
Screenshot 2024-04-17 064706

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by mohitparmar1** [comment]
#447

**Comment by mohitparmar1** [comment]
Hi @hkirat can you check this. its so basic but I'm learning for you and implementing day by day

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "husky": "^9.0.7",
     "postcss": "^8",
     "prettier": "^3.2.4",
-    "prisma": "^5.6.0",
+    "prisma": "^5.12.1",
```
Comment (Gautam-Hegde) [inline package.json:88]: why dep version changes?

Comment (mohitparmar1) [reply package.json:88]: I had some error related to Prisma so I just installed Prisma manually, any problem?

**Pull request closed without merging**